src/content/concepts/module-resolution.md

```diff
@@ -55,18 +55,18 @@ import 'module';
 import 'module/lib/file';
 ```
 
-Modules are searched for inside all directories specified in [`resolve.modules`](/configuration/resolve/#resolve-modules).
-You can replace the original module path by an alternate path by creating an alias for it using [`resolve.alias`](/configuration/resolve/#resolve-alias) configuration option.
+Modules are searched for inside all directories specified in [`resolve.modules`](/configuration/resolve/#resolvemodules).
+You can replace the original module path by an alternate path by creating an alias for it using [`resolve.alias`](/configuration/resolve/#resolvealias) configuration option.
 
 Once the path is resolved based on the above rule, the resolver checks to see if the path points to a file or a directory. If the path points to a file:
 
 - If the path has a file extension, then the file is bundled straightaway.
-- Otherwise, the file extension is resolved using the [`resolve.extensions`](/configuration/resolve/#resolve-extensions) option, which tells the resolver which extensions (eg - `.js`, `.jsx`) are acceptable for resolution.
+- Otherwise, the file extension is resolved using the [`resolve.extensions`](/configuration/resolve/#resolveextensions) option, which tells the resolver which extensions (eg - `.js`, `.jsx`) are acceptable for resolution.
 
 If the path points to a folder, then the following steps are taken to find the right file with the right extension:
 
-- If the folder contains a `package.json` file, then fields specified in [`resolve.mainFields`](/configuration/resolve/#resolve-mainfields) configuration option are looked up in order, and the first such field in `package.json` determines the file path.
-- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolve-mainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
+- If the folder contains a `package.json` file, then fields specified in [`resolve.mainFields`](/configuration/resolve/#resolvemainfields) configuration option are looked up in order, and the first such field in `package.json` determines the file path.
+- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolvemainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
 - The file extension is then resolved in a similar way using the `resolve.extensions` option.
```

Comment:
can you add link for `resolve.extensions` ->  `/configuration/resolve/#resolveextensions`